### DISCUSSION TITLE
Admin UI Sign-in/Sign-out

### DISCUSSION THREAD

**Discussion by h0byte** [contember/admin#536] [Support]
Hi,
I wanted to ask, does the Admin UI have a sign in/sign out form implemented and available to use, or do we need to implement our own Admin UI authentication?

If I missed it in the docs, would you be able to please point me to the relevant link?

Thank you kindly! :) 

EDIT: I just realised that I'm asking in a very round about way if the Admin UI (Contember interface) has authentication and/or authorization

**Comment by honzasladek** [comment]
Hi again @sammwashere! Our fault it's not in the docs, thanks for catching that! (Will resolve ASAP.) Interface does provide login / password reset / register flows, so it's there. Regarding the authentication/authorization I tried to answser in your second question: https://github.com/orgs/contember/discussions/537#discussioncomment-6182709 

Let me know if it's clearer now, happy to provide more details.
